hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/token/BlockTokenVerifier.java

```diff
@@ -76,6 +80,10 @@ protected Object getService(ContainerCommandRequestProtoOrBuilder cmd) {
   @Override
   protected void verify(OzoneBlockTokenIdentifier tokenId,
       ContainerCommandRequestProtoOrBuilder cmd) throws SCMSecurityException {
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug("Verifying token:{} for user:{} ", tokenId,
+          tokenId.getUser());
```

Comment:
Previously this message was logged after reading the token (before verifying signer cert, etc.)  Is this difference OK?

Reply:
Thanks for the review, @adoroszlai. I think the difference should be fine.